src/integrify/kapital/schemas/request.py

```diff
@@ -0,0 +1,93 @@
+from decimal import Decimal
+from typing import Dict, List, Optional
+
+from pydantic import Field
+
+from integrify.kapital import env
+from integrify.kapital.schemas.utils import BaseSchema
+from integrify.schemas import PayloadBaseModel
+
+
+class CreateOrderRequestSchema(PayloadBaseModel, BaseSchema):
+    amount: Decimal
+    currency: str
+    description: str
+    language: Optional[str] = env.KAPITAL_INTERFACE_LANG
+    hpp_redirect_url: Optional[str] = Field(default=env.KAPITAL_REDIRECT_URL)
+    type_rid: Optional[str] = Field(default='Order_SMS')
+    hpp_cof_capture_purposes: Optional[List[str]] = Field(default=['Cit'])
```

Comment:
Python 3.9 versiyasindan sonra "typing" librarydeki List, Dict, Tuple yerine birbasa builtin olan list, dict, tuple istifade etmek olur.

https://docs.python.org/3/whatsnew/3.9.html#pep-585-type-hinting-generics-in-standard-collections